e2e2/test/images/nvidia/Dockerfile

```diff
@@ -41,7 +41,6 @@ RUN apt install -y \
   cmake \
   apt-utils \
   libhwloc-dev \
-  cuda-demo-suite-$CUDA_MAJOR_VERSION-$CUDA_MINOR_VERSION \
```

Comment:
this is needed for the tests https://github.com/aws/aws-k8s-tester/blob/main/e2e2/test/images/nvidia/gpu_unit_tests/tests/test_basic.sh

Reply:
Yeah was going to change this. Updated. Thanks for the review